problems/arraysIntro.js

```diff
@@ -4,57 +4,76 @@
  * @param {string} fruit
  * @returns {string[]}
  */
-function addFruitToEnd() {}
+const addFruitToEnd = (fruits, fruit) => {
+  fruits.push(fruit);
+  return fruits;
+}
+
+
 
 /**
  * Adds a fruit to the front of an array of fruits and returns the array.
  * @param {string[]} fruits
  * @param {string} fruit
  * @returns {string[]}
  */
-function addFruitToFront() {}
+const addFruitToFront = (fruits, fruit) => {
+  console.log(fruits)
+  fruits.unshift(fruit);
+  return fruits;
+}
 
 /**
  * Removes the last fruit from an array of fruits and returns the array.
  * @param {string[]} fruits
  * @returns {string[]}
  */
-function removeLast() {}
+const removeLast = (fruits) => {
+  fruits.pop()
+  return fruits;
+}
 
 /**
  * Removes and returns the last fruit from from an array of fruits
  * @param {string[]} fruits
  * @returns {string} The fruit that was removed
  */
-function removeLastandReturnLast() {}
+const removeLastandReturnLast = (fruits) => fruits.pop()
 
 /**
  * Takes in an array and returns the length
  * @param {Array} array
  * @returns {number}
  */
-function arrayLength() {}
+const arrayLength = (array) => array.length
 
 /**
  * Removes the first fruit in an array of fruits and returns the array.
  * @param {string[]} fruits
  * @returns {string[]}
  */
-function removeFirst() {}
+const removeFirst = fruits => {
+  fruits.shift()
+  return fruits;
+}
 
 /**
  * Removes and returns the first fruit in an array of fruits.
  * @param {string[]} fruits
  * @returns {string} The fruit that was removed
  */
-function removeFirstandReturnFirst() {}
-
+const removeFirstandReturnFirst = fruits => {
+  let removed = fruits.shift()
+  return removed;
+}
 /**
  * Returns a string made by joining all of the elements in an array of fruits with a '$' character.
  * @param {string[]} fruits
  * @returns {string}
- */
-function arrayToString() {}
+ */ // [0,1,2]  =  > 0,1,2
```

Comment:
Same thing as your console.log, don't get in the habit of including your comments